api/config/index.js

```diff
@@ -1,2 +1,5 @@
-export { default as logger } from "./winstonlog";
```

Comment:
Please where are you now I'm posting Winston and Dbconnection can't seem to find it

Reply:
winston is imported in dbConnection.js 
dbconnection will be imported in the main server file

Reply:
The purpose of the index is to import and export most modules used in the folder
So we would not have too many lines of import at that server, to keep it brief.